<ui version="4.0" >
 <class>SettingsHttpForm</class>
 <widget class="QWidget" name="SettingsHttpForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>HTTP/FTP</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Default proxy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="comboDefaultProxy" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Automatic user credentials</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>If new download's URI matches a certain regexp, defined user credentials will be set.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QPushButton" name="pushAuthAdd" >
        <property name="text" >
         <string>Add</string>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QPushButton" name="pushAuthEdit" >
        <property name="text" >
         <string>Edit</string>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QPushButton" name="pushAuthDelete" >
        <property name="text" >
         <string>Delete</string>
        </property>
        <property name="icon" >
         <iconset/>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="3" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item rowspan="4" row="0" column="0" >
       <widget class="QListWidget" name="listAuths" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listAuths</tabstop>
  <tabstop>pushAuthAdd</tabstop>
  <tabstop>pushAuthEdit</tabstop>
  <tabstop>pushAuthDelete</tabstop>
 </tabstops>
 <resources>
  <include location="../gfx/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>listAuths</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>pushAuthEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>209</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel" >
     <x>475</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
